packages/wrangler/src/inspect.ts

```diff
@@ -151,6 +151,12 @@ export default function useInspector(props: InspectorProps) {
 			);
 			ws.close(1013, "Too many clients; only one can be connected at a time");
 		} else {
+			remoteWebSocket?.send(
+				JSON.stringify({
+					id: 100_000,
+					method: "Debugger.disable",
+				})
+			);
```

Comment:
Is there any feasible way to test this? I won't block on it since I am not sure myself, the manual testing went well too.

Reply:
Unfortunately not... I manually tested it a fair bit and it seemed to work—if you're able to do that as well (with the new devtools), I think that's as much confidence as we can get